evalml/data_checks/outliers_data_check.py

```diff
@@ -42,17 +41,18 @@ def validate(self, X, y=None):
 
         X = infer_feature_types(X)
         X = X.ww.select("numeric")
+        X.ww.drop(X.columns[X.isna().all()].tolist(), inplace=True)
 
         if len(X.columns) == 0:
             return results
 
         has_outliers = []
         for col in X.columns:
-            outlier_results = OutliersDataCheck._outlier_score(X[col], False)
-            if (
-                outlier_results is not None and outlier_results["score"] <= 0.9
-            ):  # 0.9 is threshold indicating data needs improvement
+            col_series = ww.init_series(X[col])
```

Comment:
You can get this series with less overhead by doing `col_series = X.ww[col]`. It'll retain all the typing information that's already present, avoiding type inference

Reply:
Thanks!